Makefile

```diff
@@ -177,10 +177,10 @@ test-ui: lint-ui test-ui-unit test-integration # Run frontend tests
 test-ui-unit: # Run frontend jest unit tests
 	yarn test:unit
 
-test-integration: # Run frontend cypress integration tests
+test-integration: install-examples # Run frontend cypress integration tests
 	yarn test:integration
 
-test-integration-debug: # Open cypress integration test debugger
```

Comment:
Feedback wanted here: not sure if this is the best solution, but it's a relatively easy fix

Reply:
Not sure if I'm who you're looking for feedback from, but it seems fine to me? Maybe add a comment to explain why though